Lib/test/test_inspect.py

```diff
@@ -167,12 +167,33 @@ def test_iscoroutine(self):
         gen_coro = gen_coroutine_function_example(1)
         coro = coroutine_function_example(1)
 
+        class PMClass:
```

Comment:
How about a test case for a partialmethod wrapping a normal method?

Reply:
Testing how inspect interacts with `partialmethod()` on regular methods is [covered elsewhere already](https://github.com/python/cpython/blob/9a669d58e8cb586fba38c84d5b631cd8a95d0c0c/Lib/test/test_inspect.py#L859-L871); these are the coroutine-specific tests.